src/naturallanguage.cs

```diff
@@ -485,6 +522,54 @@ interface NLEmbedding {
 		[Static]
 		[Wrap ("Write (dictionary.GetDictionary ()!, language.HasValue ? language.Value.GetConstant () : null, revision, url, out error)")]
 		bool Write (NLVectorDictionary dictionary, NLLanguage? language, nuint revision, NSUrl url, [NullAllowed] out NSError error);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Export ("currentSentenceEmbeddingRevisionForLanguage:")]
+		nuint GetCurrentSentenceEmbeddingRevision (NSString language);
+
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Wrap ("GetCurrentSentenceEmbeddingRevision (language.GetConstant ()!)")]
+		nuint GetCurrentSentenceEmbeddingRevision (NLLanguage language);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Export ("sentenceEmbeddingForLanguage:")]
+		[return: NullAllowed]
+		NLEmbedding GetSentenceEmbedding (NSString language);
+
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Wrap ("GetSentenceEmbedding (language.GetConstant ()!)")]
+		[return: NullAllowed]
+		NLEmbedding GetSentenceEmbedding (NLLanguage language);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Export ("sentenceEmbeddingForLanguage:revision:")]
+		[return: NullAllowed]
+		NLEmbedding GetSentenceEmbedding (NSString language, nuint revision);
+
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Wrap ("GetSentenceEmbedding (language.GetConstant ()!, revision)")]
+		[return: NullAllowed]
+		NLEmbedding GetSentenceEmbedding (NLLanguage language, nuint revision);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Export ("supportedSentenceEmbeddingRevisionsForLanguage:")]
+		NSIndexSet SupportedSentenceEmbeddingRevisions (NSString language);
```

Comment:
I think this needs a verb:

```suggestion
		NSIndexSet GetSupportedSentenceEmbeddingRevisions (NSString language);
```